From 087138b3641c8feeddc7e536e19a82d25bbb291e Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Tue, 28 Aug 2018 20:32:58 +0300
Subject: [PATCH 03/10] vold: Move QCOM HW FDE inclusion under Lineage
 namespace

 * Match the new Lineage naming convention, while at it.

Change-Id: If6a7a0c159ccf4ffed4c5946bf8be6ad04b6665a
---
 Android.bp | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/Android.bp b/Android.bp
index 64b29c2..30b0367 100644
--- a/Android.bp
+++ b/Android.bp
@@ -108,6 +108,8 @@ cc_library_static {
     defaults: [
         "vold_default_flags",
         "vold_default_libs",
+        "vold_hw_fde_defaults",
+        "vold_hw_fde_perf_defaults",
     ],
 
     srcs: [
@@ -163,14 +165,6 @@ cc_library_static {
         debuggable: {
             cppflags: ["-D__ANDROID_DEBUGGABLE__"],
         },
-        device_support_hwfde: {
-            cflags: ["-DCONFIG_HW_DISK_ENCRYPTION"],
-            header_libs: ["libcryptfs_hw_headers"],
-            shared_libs: ["libcryptfs_hw"],
-        },
-        device_support_hwfde_perf: {
-            cflags: ["-DCONFIG_HW_DISK_ENCRYPT_PERF"],
-        },
     },
     shared_libs: [
         "android.hardware.health.storage@1.0",
@@ -191,6 +185,7 @@ cc_binary {
     defaults: [
         "vold_default_flags",
         "vold_default_libs",
+        "vold_hw_fde_defaults",
     ],
 
     srcs: ["main.cpp"],
@@ -226,9 +221,6 @@ cc_binary {
                 "libarcvolume",
             ],
         },
-        device_support_hwfde: {
-            shared_libs: ["libcryptfs_hw"],
-        }
 	},
 }
 
-- 
2.30.2

